packages/hunspell-reader/src/__snapshots__/affToDictInfo.test.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`affToDictInfo parseAffFile 1`] = `
 {
-  "accents": "\\u0300-\\u0303\\u0308\\u0327",
+  "accents": "̀-̧̃̈",
```

Comment:
Maybe a bug with the snapshotting, but there is now accents on the closing quote

Reply:
The actual value is a bunch of accents. It just doesn't render correctly.

Reply:
@nschonni,

Thank you for checking.